docs/source/userdoc.rst

```diff
@@ -483,6 +483,40 @@ Re-parsing with another parser:
 .. seealso:: :py:mod:`ftw.testbrowser.core.Browser.parse`
 
 
+HTTP requests
+===============
+
+`ftw.testbrowser` also supports not following redirects. This is useful for
+testing the bodies of redirect responses or inspecting `Location` headers.
+
+This is currently not implemented for mechanize.
+
+.. code:: py
+
+    from ftw.testbrowser import browsing
+    from ftw.testbrowser.tests.alldrivers import all_drivers
+    from ftw.testbrowser.tests.alldrivers import skip_driver
+    from unittest2 import TestCase
+
+
+    @all_drivers
+    class TestRedirects(TestCase):
+
+      @browsing
+      def test_redirects_are_followed_automatically(self, browser):
+          browser.open(view='test-redirect-to-portal')
+          self.assertEquals(self.portal.absolute_url(), browser.url)
+          self.assertEquals(('listing_view', 'plone-site'), plone.view_and_portal_type())
+
+      @skip_driver(LIB_MECHANIZE, 'Changing redirect following behavior is not supported.')
```

Comment:
I suggest to remove `@all_drivers` and `@skip_driver` from the documentation as those are testbrowser internal features which may confuse a "user" of the testbrowser.